lib/deppack.js

```diff
@@ -183,9 +191,13 @@ const xBrowserResolve = (mod, opts, cb) => {
         err = isGlob ? `Could not load global module '${mod}'.` : `Could not load module '${mod}' from '${src}'.`;
 
         if (packages[topLevel] == null) {
-          err = err += ` Possible solution: add '${topLevel}' to package.json and \`npm install\`.`;
+          err += ` Possible solution: add '${topLevel}' to package.json and \`npm install\`.`;
         } else {
-          err = err += ` Possible solution: run \`npm install\`.`;
+          if (overrides[mod]) {
+            err += ` Possible solution: check your overrides in package.json.`;
```

Comment:
@goshakkk btw, you don't need to re-run npm install after overriding smth. 
What do you think about:
```javascript
err += ` Possible solution: ${overrides[mod] ? 'check your overrides in package.json; ' : ''}run \`npm install\`. `;
```

Or we can also try to check if main file of package is found in `applyPackageOverride` function (and throw error if file not found)